test/e2e_node/podresources_test.go

```diff
@@ -48,7 +48,7 @@ type podDesc struct {
 	cntName        string
 	resourceName   string
 	resourceAmount int
-	cpuCount       int
+	cpuRequest     float32
```

Comment:
why? I think we can express the cpuamount in millicores so we can keep the `int`.
This way, say, 0.5 cpus becomes `500` and 2.5 becomes `2500`. If we go this path, let's add a comment documenting the request is in millicores (not whole cpus as it was).

Reply:
Sure, that sounds reasonable to me. Will update the e2e tests soon.